packages/flutter_tools/lib/src/build_system/build_system.dart

```diff
@@ -369,6 +372,8 @@ class Environment {
   /// which prevents the config from leaking into different builds.
   final Map<String, String> defines;
 
+  final List<String> dartDefines;
```

Comment:
This should be a key under `defines` so that rebuilds are correct

Reply:
Done.